docs/source/te/_toctree.yml

```diff
@@ -0,0 +1,8 @@
+- sections:
+  - local: index
+    title: 🤗 Transformers
+  - local: quicktour
+    title: త్వరిత పర్యటన
+  - local: installation
+    title: సంస్థాపన
```

Comment:
The doc-builder is failing because this file does not exist yet.

```suggestion
```

Reply:
Thanks for bringing that to my attention. I'll continue it in my future contributions.